src/pages/index.js

```diff
@@ -3,95 +3,157 @@ import Card from "/src/components/Card.js";
-
-const enableValidation = (config) => {
-  const formList = [...document.querySelectorAll(config.formSelector)];
-  formList.forEach((form) => {
-    const validator = new FormValidator(config, form);
-    const formName = form.getAttribute("name");
+import * as constants from "/src/utils/constants.js";
+import Api from "/src/components/Api.js";
+// User Data
+const userInfo = new UserInfo({
+  userName: constants.profileName,
+  userDescription: constants.profileDescription,
+  userProfileImage: constants.profileAvatar,
+});
+// Api Instantiation
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "9fbfab1b-69d6-43a9-b085-60d57b074309",
+    "Content-Type": "application/json",
+  },
+});
 
-    formValidators[formName] = validator;
-    validator.enableValidation();
-  });
-};
-const createCard = (data) => {
-  const cardElement = new Card({
-    data: data,
-    cardSelector: cardTemplate,
-    handleCardClick: () => {
-      popupImage.open(data);
+Promise.all([api.getUserInformation(), api.getInitialCards()]).then(
+  ([userData, initialCards]) => {
+    userInfo.setUserInfo(userData);
+    userInfo.setUserProfileImage(userData);
+    const apiCardSection = new Section(
+      {
+        items: initialCards,
+        renderer: addCard,
+      },
+      constants.cardGallery
+    );
+    apiCardSection.renderItems();
+  }
+);
+// Create Card + Add Card Functions
+const createCard = (cardData) => {
+  const cardElement = new Card(
+    cardData,
+    {
+      handleCardClick: () => popupImage.open(cardData),
+      handleDeleteClick: () => {
+        deleteModalNew.open();
+        deleteModalNew.submitForm(() => {
+          api
+            .deleteCard(cardData._id)
+            .then(() => {
+              deleteModalNew.close();
+              cardElement.removeCard();
+            })
+            .catch((err) => console.error(err));
+        });
+      },
+      handleLikeClick: (card) => {
+        // debugger;
+        if (!cardData.isLiked) {
+          api
+            .addLike(card)
+            .then((res) => {
+              cardElement.setLike(res);
+              console.log("add");
+              console.log(res.isLiked);
+            })
+            .catch((err) => console.error(err));
+        } else {
+          api
+            .deleteLike(card)
+            .then((res) => {
+              cardElement.setLike(res.isLiked);
+              console.error("delete");
+              console.log(res.isLiked);
+            })
+            .catch((err) => console.error(err));
+        }
+      },
     },
-  });
+    constants.cardTemplate
+  );
   return cardElement.getView();
 };
-const addCard = (data) => {
-  const card = createCard(data);
+const addCard = (cardData) => {
+  const card = createCard(cardData);
   cardSection.setItem(card);
 };
-
-const popupImage = new PopupWithImage(
+// Section instance to use methods
+const cardSection = new Section(
```

Comment:
Only one instance of class `Section` should be created in the project because there is only **one** section for the cards.